tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -69,13 +77,11 @@ $array = [
     /* comment */
     /* comment */
     /* comment */
-    "foo" => "bar"
-    /* comment */
-    /* comment */, /* comment */
+    "foo" => /* comment */ /* comment */
```

Comment:
This doesn't look fantastic but I think its worth the trade-off for now of not breaking the new example

Reply:
Yep, let's fix it in future :+1: